Solution for fix "fatal error: 'ext/standard/php_mt_rand.h' file not found"

In newer php version in compilation process parallel I see error:
```
/tmp/parallel-1.2.1/src/poll.c:23:10: fatal error: 'ext/standard/php_mt_rand.h' file not found
#include "ext/standard/php_mt_rand.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [src/poll.lo] Error 1
```

For fix this error replace string **ext/standard/php_mt_rand.h** to **ext/random/php_random.h** in file **src/poll.c**

After this modification parallel successfully compiled in any latest php versions! :-)

_**P.S. Developer, please fix this error for newer php versions!**_

--- a/src/poll.c
+++ b/src/poll.c
@@ -20,7 +20,11 @@
 
 #include "parallel.h"
 
+#if PHP_VERSION_ID >= 80400
+#include "ext/random/php_random.h"
+#else
 #include "ext/standard/php_mt_rand.h"
+#endif
 
 typedef struct _php_parallel_events_poll_t {
     struct timeval stop;

